readme.md

```diff
@@ -148,6 +148,10 @@ The attribute should be an image URL:
 <div class="pinterest" data-media="https://placekitten.com/200/400">Pin</div>
 ```
 
+### VK
```

Comment:
This is in the middle of the “Pinterest” section :–)
Let’s place this section after Twitter, OK? VK is more important than Telegram or Pinterest but less important than Twitter.